stl/inc/chrono

```diff
@@ -5315,7 +5319,23 @@ namespace chrono {
         // chrono-spec
         while (_Begin != _End && *_Begin != '}') {
             if (*_Begin == '%') { // conversion-spec
-                _Begin = _Parse_conversion_specs(_Begin, _End, _Callbacks);
+                _CharT _Next_ch = *(_Begin + 1);
+                switch (_Next_ch) {
+                case 'n':
+                    _Callbacks._On_lit_char('\n');
+                    ++_Begin;
+                    break;
+                case 't':
+                    _Callbacks._On_lit_char('\t');
+                    ++_Begin;
+                    break;
+                case '%':
+                    _Callbacks._On_lit_char('%');
+                    ++_Begin;
+                    break;
+                default: // some other type
+                    _Begin = _Parse_conversion_specs(_Begin, _End, _Callbacks);
+                }

```

Comment:
Nitpick: We conventionally `break;` after every `case`/`default` including the last one. Although this is technically unnecessary, it makes the code more robust for later refactorings when cases are rearranged or added.